r_packages.csv

```diff
@@ -1,5 +1,6 @@
 name,version,sha256
 BH,1.72.0-3,888ec1a3316bb69e1ba749b08ba7e0903ebc4742e3a185de8d148c13cddac8ab
+DataCombine,0.2.21,352b235612e2cf8234b3ab5f9aa6f7a394b006b98d24e315940ccc65c4218b47
```

Comment:
no readr in this file?

Reply:
```
myproject) emersonm@emersonm:~/toxicity-detector$ bazel run main:factor
DEBUG: Rule 'com_grail_rules_r' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "705c63ffc8e7ab4c9626d2be658d6b55e803d9133e8ef0384d1049b13b8bfd99"
DEBUG: Repository com_grail_rules_r instantiated at:
  no stack (--record_rule_instantiation_callstack not enabled)
Repository rule http_archive defined at:
  /usr/local/google/home/emersonm/.cache/bazel/_bazel_emersonm/944306df6cbfee92be3237efb3eb9146/external/bazel_tools/tools/build_defs/repo/http.bzl:336:31: in <toplevel>
INFO: Analyzed target //main:factor (0 packages loaded, 0 targets configured).
INFO: Found 1 target...
Target //main:factor up-to-date:
  bazel-bin/main/factor
INFO: Elapsed time: 0.366s, Critical Path: 0.05s
INFO: 0 processes.
INFO: Build completed successfully, 1 total action
INFO: Build completed successfully, 1 total action
Error in library(readr) : there is no package called ‘readr’
Execution halted
(myproject) emersonm@emersonm:~/toxicity-detector$ 

```

Reply:
Hmmmm... I can see `readr` in line 53 in this file.

Reply:
Hmmmm.. I can see `readr` in line 53

Reply:
So it is. Still, I blaze ran again and got the same error message. I don't see readr in the built libraries:

```
(myproject) emersonm@emersonm:~/toxicity-detector/bazel-bin/main/factor.runfiles$ ls
__main__  MANIFEST  R_assertthat  R_cli  R_crayon  R_DataCombine  R_data_table  R_digest  R_dplyr  R_ellipsis  R_fansi  R_generics  R_glue  R_lifecycle  R_magrittr  R_pillar  R_pkgconfig  R_purrr  R_R6  R_rlang  R_tibble  R_tidyselect  R_utf8  R_vctrs
```

Does it work (at least get through the imports) when you run it through "blaze run"?  If so, what built libraries do you get?

Reply:
OH! I realized the problem - I've been running it through `feed_data` binary, which has all the dependencies declared in the `BUILD` file. Sorry about that~ I've pushed the updated `BUILD` file